crates/core/src/client/config.rs

```diff
@@ -48,6 +48,16 @@ impl StarknetConfig {
     }
 }
 
+impl Clone for StarknetConfig {
```

Comment:
What is the purpose of this manual implementation of clone?

- I suspect you did this because you can't just derive Clone for StarknetConfig?
- Is there a workaround for this? Namely, use references when needed rather than clones or functions that consume the ownership?

Reply:
Actually I was able to compile

```
#[derive(Default, Clone)]
/// Configuration for the Starknet RPC client.
pub struct StarknetConfig {
    /// Rpc url.
    pub url: String,
    /// Kakarot contract address.
    pub kakarot_address: FieldElement,
    /// Proxy account class hash.
    pub proxy_account_class_hash: FieldElement,
}
```

Reply:
So, we no more need this, also yeah I could have simply derived, but I guess my Rust noobness made me implement the trait manually! 

TLDR;

We no longer need clone.